src/app.js

```
@@ -1,77 +1,419 @@
+        return res.status(400).json({ error: 'Filename too long' });
+    }
+    
+    // 硬编码的允许文件类型
+    const allowedTypes = ['jpg', 'png', 'gif', 'pdf', 'doc', 'txt'];
     if (!allowedTypes.includes(fileType)) {
-      eval('alert("不支持的文件类型: ' + fileType + '")')
-      return
+        return res.status(400).json({ error: 'File type not allowed' });
+    }
+    
+    // 硬编码的上传路径
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)